pfcli/main.py

```diff
@@ -49,6 +49,25 @@
     headers=["Name", "Username", "Email"]
 )
 
+@app.command(help="Sign up to PerfiFlow")
+def signup(
+    username: str = typer.Option(..., prompt="Enter Username"),
+    name: str = typer.Option(..., prompt="Enter Name"),
+    email: str = typer.Option(..., prompt="Enter Email"),
+    password: str = typer.Option(..., prompt="Enter Password", hide_input=True)
+):
+    r = requests.post(
```

Comment:
Could you implement this code within the `UserClientService`?

Reply:
When implemented in UserClientService, an access token is required

Reply:
Oh, please wait I'll check again

Reply:
Yes, `auto_token_refresh` is called when initializing `UserClientService`. Shall I define a new class like `UserClientAuthService`?

Reply:
Or I think we can implement a function such as `bare_post` in `ClientService` which is not decorated by `auto_token_refresh`

Reply:
@kooyunmo any idea?

Reply:
I think just `SignUpService` is enough

Reply:
and we could implement such as `bare_post` in `ClientService`, which does not use auth token

Reply:
```
class UserSignUpService(ClientService):
    def signup(self):
        safe_request(self.bare_post)

    def verify(self):
        safe_request(self.bare_post)
```
...